tests/python/unittest/test_gluon_rnn.py

```diff
@@ -25,6 +25,7 @@
 
 def test_rnn():
     cell = gluon.rnn.RNNCell(100, prefix='rnn_')
+    print(cell)
```

Comment:
Are these prints intentional?

Reply:
Yes, this is for verifying whether cell `__repr__` works.